src/main/cluster.ts

```diff
@@ -47,6 +47,7 @@ export type ClusterPreferences = {
     service: string;
     port: number;
   };
+  prometheusSource?: string;
```

Comment:
Just thinking if this should be an object? It would be a bit more future-proof, for example if we allow custom prometheus "adapters" from fs path / url.

Reply:
It's now object `prometheusProvider: { type: string }`